resources/public/api/conf/1.0/amend_relief_investments.raml

```diff
@@ -0,0 +1,38 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearNotSupported
+  - errors.formatValue
+  - errors.incorrectOrEmptyBody
+  - errors.clientOrAgentNotAuthorised
+  - errors.formatDateOfInvestment
+  - errors.formatUniqueInvestmentReference
+  - errors.formatName
+  - errors.notFound
+displayName: Amend relief investments
```

Comment:
The display name should have '[test only] at the end, also best to put capitals at the start of each of the words.